### PR TITLE
To do list structure 

### PR DESCRIPTION
**Pull request by Rachelwebdev**
### In this milestone, I :

- Set up webpack to bundle javascript codes
- Set up Linters for HTML, CSS and JavaScript
- Divided my files into modules
- Created an array of objects of list items
- Created a function to  iterate over them for each item in the list
- Rendered the page dynamically on reload

### CODE REVIEW

File: src/modules/listCard.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const cardDisplay = () => {
+  listItems.forEach((item) => {
+    const toDoListItems = `
+        <ul>
+          <li class="item-list">${item.description}</li>         
+        </ul>
+          `;
+    toDoListContainer.insertAdjacentHTML('afterbegin', toDoListItems);
+  });
+};
```
Comment (ShadyShawkat) [inline src/modules/listCard.js]: - [ ] Kindly make sure that the todo list is displayed in the order of their indexes, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Rachelwebdev, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Rachelwebdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/listCard.js (hunk 1 above)
Comment (Rachelwebdev) [reply src/modules/listCard.js]: Noted, I would ensure this is worked on. Thanks for the review

**Comment by Rachelwebdev** [comment]
Thanks for the Approval 👍

**Pull request merged**